### PR TITLE
Ndlano/issues#389 style improvements

### PR DESCRIPTION
**Pull request by sebastianjg**
- Added Accordion component
- Replaced existing accordion with new design
- Added content-accordion
- Some style adjustments

### CODE REVIEW

File: src/components/Accordion.jsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  hidden,
+  handleToggle,
+  className,
+  ...rest
+}) => {
+  const modifiers = {
+    fill,
+  };
+
+  const accordionClasses = hidden ? classes('hidden') : classes('visible');
```
Comment (chrpeter) [inline src/components/Accordion.jsx]: Dette er vel strengt talt en modifier?

File: src/components/Fields.jsx [hunk 2]
```diff
@@ -164,15 +164,28 @@ TextField.defaultProps = {
   noBorder: false,
 };
 
-export const FieldDescription = ({ children }) =>
-  <p {...classes('description')}>
-    {children}
-  </p>;
+export const FieldDescription = ({ obligatory, children }) => {
+  const fieldDescriptionClasses = obligatory
+    ? classes('description', 'obligatory')
```
Comment (chrpeter) [inline src/components/Fields.jsx]:
```javascript
classes('description', obligatory ? 'obligatory' : '')
```
Funker dette? Vet ikke hva jeg liker best. Hva tenker du?

Comment (sebastianjg) [inline src/components/Fields.jsx]: Funker den, liker den bedre 👍

File: src/components/Accordion.jsx (hunk 1 above)
Comment (sebastianjg) [inline src/components/Accordion.jsx]: Oppdatert

File: src/components/Accordion.jsx [hunk 3]
```diff
@@ -29,8 +29,11 @@ const Accordion = ({
   const modifiers = {
     fill,
   };
+  const contentModifiers = {
+    hidden,
+    visible: !hidden,
+  };
 
-  const accordionClasses = hidden ? classes('hidden') : classes('visible');
   const buttonClasses = fill ? classes('button', 'fill') : classes('button');
```
Comment (chrpeter) [inline src/components/Accordion.jsx]: `fill ? fill : ''`

**Pull request merged**